core/node/events/migrations/snapshot_migration_0002.go

```diff
@@ -0,0 +1,25 @@
+package migrations
+
+import (
+	. "github.com/river-build/river/core/node/protocol"
+	"github.com/river-build/river/core/node/shared"
+)
+
+func snapshot_migration_0002(iSnapshot *Snapshot) *Snapshot {
+	switch snapshot := iSnapshot.Content.(type) {
+	case *Snapshot_SpaceContent:
+		for _, channel := range snapshot.SpaceContent.Channels {
+			channelId, err := shared.StreamIdFromBytes(channel.ChannelId)
+			if err != nil {
+				panic(err)
+			}
+			if shared.IsDefaultChannelId(channelId) {
+				channel.Autojoin = true
+			} else {
+				channel.Autojoin = false
+			}
+			channel.ShowUserJoinLeaveEvents = true
```

Comment:
i suggest naming this HideuserJoinLeaveEvents so that the default value is what we want by default.